src/common/components/InputField/PrefixIcon/PrefixIcon.tsx

```diff
@@ -0,0 +1,10 @@
+// import styles from "./PrefixIcon.module.scss";
+import SearchIcon from "@/images/ic_search.svg";
+
+interface PrefixIconProp {
+  prefix: string;
```

Comment:
지금 구조라면, 이런식으로 표현되는게 더 좋을 것 같습니다~
```suggestion
  iconName: 'search';
```